.travis.yml

```diff
@@ -1,58 +1,101 @@
-# make it explicit that we favor the new container-based travis workers
+# Make it explicit that we favor the
+# new container-based Travis workers
 language: python
+dist: xenial
 
 cache:
   apt: true
   directories:
-  - $HOME/.cache/pip
-  - $HOME/.ccache
-
-dist: xenial
+    - $HOME/.cache/pip
+    - $HOME/.ccache
 
 env:
   global:
-    # Directory where tests are run from
-    - TEST_DIR=/tmp/sklearn
+    - CPU_COUNT=3
+    - TEST_DIR=/tmp/sklearn  # Test directory for continuous integration jobs
+    - PYTEST_VERSION=latest
     - OMP_NUM_THREADS=2
     - OPENBLAS_NUM_THREADS=2
-    - PYTEST_VERSION=latest
+    - SKLEARN_BUILD_PARALLEL=3
+    - SKLEARN_SKIP_NETWORK_TESTS=1
+    # Custom environment variables for the ARM wheel builder
+    - CIBW_BUILD_VERBOSITY=1
+    - CIBW_TEST_REQUIRES="pytest pytest-xdist threadpoolctl"
+    - CIBW_TEST_COMMAND="bash {project}/build_tools/travis/test_wheels.sh"
+    - CIBW_ENVIRONMENT="CPU_COUNT=8
+                        OMP_NUM_THREADS=2
+                        OPENBLAS_NUM_THREADS=2
+                        SKLEARN_BUILD_PARALLEL=8
+                        SKLEARN_SKIP_NETWORK_TESTS=1"
+    # Nightly upload token and staging upload token are set in Travis settings
+    - SCIKIT_LEARN_NIGHTLY_UPLOAD_TOKEN=__token__
+    - SCIKIT_LEARN_STAGING_UPLOAD_TOKEN=__token__
```

Comment:
Actually, this is wrong, no? This should be secret environment variable, not the `__token__` constant string.

Reply:
I have used the example provided by `cibuildwheel`: https://github.com/joerick/cibuildwheel/blob/49077c2a36f35d23183f49126c6c0652ea87f206/examples/travis-ci-deploy.yml

But they have recently updated the example using the secret environment variable: https://github.com/joerick/cibuildwheel/blob/d83daa3958a9809f0b03fe125ce28d1f52ab7f14/examples/travis-ci-deploy.yml

Reply:
Indeed, `travis` provides a `deploy` step, but it does not provide support for Anaconda: https://docs.travis-ci.com/user/deployment/.

Reply:
I will open a PR to remove these lines and directly use the `travis` tokens provided `Anaconda`.